### PR TITLE
all: add synopsis for the packages

### PR DESCRIPTION
**Pull request by fhs**

### CODE REVIEW

File: cblas/blas.go [hunk 1]
```diff
@@ -4,7 +4,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package cblas implements the blas interfaces.
+// Package cblas is a binding to a C BLAS library.
+// It implements the blas interfaces.
```
Comment (kortschak) [inline cblas/blas.go]:
```
// Package cblas provides bindings to a C BLAS library.
```

(Drop second line).

File: cblas/genBlas.pl [hunk 2]
```diff
@@ -40,7 +40,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package cblas implements the blas interfaces.
+// Package cblas is a binding to a C BLAS library.
```
Comment (kortschak) [inline cblas/genBlas.pl]: Same here - well, only here and regen.

File: testblas/level1double.go [hunk 3]
```diff
@@ -1,5 +1,8 @@
-// Package for testing the blas implementation
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
 
+// Package testblas tests a blas implementation.
```
Comment (kortschak) [inline testblas/level1double.go]: Package testblas provides tests for blas implementions.

**Comment by fhs** [comment]
PTAL

**Comment by kortschak** [comment]
LGTM

**Pull request merged**